src/Utils/utils.ts

```diff
@@ -544,3 +544,58 @@ export const fahrenheitToCelsius = (fahrenheit: number) => {
 export const keysOf = <T extends object>(obj: T) => {
   return Object.keys(obj) as (keyof T)[];
 };
+
+// Capitalize the first letter of each word in a string, handling edge cases
+export const startCase = (str: string): string => {
+  if (!str) return "";
+
+  return str
+    .toLowerCase()
+    .replace(/\s+/g, " ")
+    .trim()
+    .split(" ")
```

Comment:
I modified validateName to accept accented characters, hyphens and apostrophes, so let's use .split(/[\s'-]+/) here instead.

Ex: Capitalizing françois-o'neil -> François-O'Neil